designer/README.md

```diff
@@ -46,6 +46,11 @@ Base URLs should should include protocol, hostname, port number, e.g. `http://lo
 | SESSION_COOKIE_TTL                | server-side storage expiration time for sessions - in milliseconds                      | yes      |         |                             |
 | SESSION_TTL                       | server-side storage expiration time - in milliseconds                                   | yes      |         |                             |
 | USE_SINGLE_INSTANCE_CACHE         | If true, disables the redis cluster connection and uses a single node.                  | yes      |         |                             |
+| http_proxy                        | HTTP proxy to use, e.g. the one from CDP. Currently used for Hapi Wreck.                | no       |         |                             |
```

Comment:
Uppercase for consistency? Both are supported
https://www.npmjs.com/package/proxy-from-env#environment-variables

Reply:
Really glad you got this working 💪

+1 for uppercase from me too, looks good